plugins/modules/fusion_volume.py

```diff
@@ -326,9 +326,12 @@ def update_size(module, fusion, current, patches):
 
 def update_protection_policy(module, fusion, current, patches):
     wanted = module.params
+    current_policy = [
+        current.protection_policy.name if current.protection_policy else None
+    ]
```

Comment:
Why the '[]' brackets? Now `current_policy` is a list with one element. Or do I miss something?

Reply:
That should have been parentheses, strangely that did not impact the logic (the double quotes did)